docs/docker.rst

```diff
@@ -0,0 +1,92 @@
+=============
+Docker
+=============
+
+.. highlight:: sh
+
+OpenTAXII can also be run using docker. This guide assumes that you have access to a local or remote docker server, and won't go into the setup of docker.
+
+To get a default (development) instance using docker
+
+
+.. code-block:: shell
+
+    docker run -d -p 9000:9000 intelworks/opentaxii
+
+.. note::
+
+    OpenTAXII is now accessible through port 9000, and is not configured.
```

Comment:
Hmm .. Ok. will make the base a bit more ‘complete’, and move it out of the ‘example’ one.

> On 07 May 2015, at 14:40, Sergey Polzunov <notifications@github.com> wrote:
> 
> In docs/docker.rst <https://github.com/Intelworks/OpenTAXII/pull/20#discussion_r29847557>:
> 
> > +=============
> > +
> > +.. highlight:: sh
> > +
> > +OpenTAXII can also be run using docker. This guide assumes that you have access to a local or remote docker server, and won't go into the setup of docker.
> > +
> > +To get a default (development) instance using docker
> > +
> > +
> > +.. code-block:: shell
> > +
> > +    docker run -d -p 9000:9000 intelworks/opentaxii
> > +
> > +.. note::
> > +
> > +    OpenTAXII is now accessible through port 9000, and is not configured.
> Yeah, we can leave out the user (then services need to be configured with authentication set to false) or configure a default one (test/test). I think a default user is nicer - shows auth features as well
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Intelworks/OpenTAXII/pull/20/files#r29847557>.
>